conf.d/__async_prompt.fish

```diff
@@ -11,9 +11,12 @@ function __async_prompt_setup_on_startup --on-event fish_prompt
     end
 
     for func in (__async_prompt_config_functions)
+        set -U $__async_prompt_var'_'$func
         function $func -V func
-            test -e $__async_prompt_tmpdir'/'$fish_pid'_'$func
-            and cat $__async_prompt_tmpdir'/'$fish_pid'_'$func
+            if set -q $__async_prompt_var'_'$func
+                set -l result $__async_prompt_var'_'$func
+                echo -n $$result
```

Comment:
I'm using starship and this causes the multiline prompts to merge into a single line. 
I changed this to `printf "%s\n" $$result` and seems to work for both single and multiline prompts